frontend/src/components/ErrorFallback.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Alert, Row, Col } from "antd";
+import styled from "styled-components";
+
+const FallbackWrapper = styled.div`
+    margin: 200
+`;
+
+export const ErrorFallback = () => {
```

Comment:
Can we have a reload button? Which causes the crashed components to mount again, this can be useful when an error occurs on a specific event like click or state change.